Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp

```diff
@@ -90,19 +91,24 @@ static ErrorOr<String> include_path_for(StringView class_name)
         return String::formatted("<Lib{}.h>", pathed_name);
 
     // We assume that all other paths are within the current application, for now.
-    return String::formatted("<Applications/{}.h>", pathed_name);
+    // To figure out what kind of userland program this is (application, service, ...) we consider the path to the original GML file.
```

Comment:
In the future it would be nice to not make such assumptions about the project path, e.g. when using from within serenity or without the serenity source tree on hand. (third party apps etc). The build system should be able to pass this sort of information in.